src/Node/ChainInfo.ts

```diff
@@ -30,13 +30,71 @@ export default class ChainInfo {
     };
   }
 
+  /**
+   * Mapping of origin dev chain name against chain id.
+   */
+  public static get devOriginChainInfo(): any {
+    return {
+      origin: '1515', // 1515 is the dev origin chain id
```

Comment:
I am not clear why we need two identifiers for a single dev origin chain.  Can't it be single devOrigin chain identifier? I might be missing here reasoning.

Also can we identify dev chains with identifier devChain and devAuxiliary.
Identifiers origin, dev, auxiliary are confusing and very generic.